src/test/java/uk/gov/cslearning/catalogue/api/CourseControllerTest.java

```diff
@@ -265,24 +265,46 @@ public void shouldConcatenateMultipleParameters() throws Exception {
                 .andExpect(jsonPath("$.results[0].id", equalTo(course.getId())));
     }
 
-
     @Test
     public void shouldListMandatoryCourses() throws Exception {
+
         String department = "department1";
         String status = "Published";
+        String type = "REQUIRED_LEARNING";
 
-        Course course = new Course();
+        List<String> organisationParents = new ArrayList();
+        organisationParents.add("dh");
+
+        String dh = "dh";
+        String hmrc = "hmrc";
+        Set<String> dh_hmrc_departments = new HashSet<>(Arrays.asList(dh, hmrc));
+
+        Audience dh_hmrc_audiences = new Audience();
+        dh_hmrc_audiences.setDepartments(dh_hmrc_departments);
+        dh_hmrc_audiences.setType(Audience.Type.REQUIRED_LEARNING);
+
+        Set<Audience> audiences1 = new HashSet<>(Arrays.asList(dh_hmrc_audiences));
+
+        Course course1 = new Course();
+        course1.setAudiences(audiences1);
+
+        List<Course> expected = new ArrayList<>();
+        expected.add(course1);
+
+        when(courseService.getOrganisationParents(department))
+                .thenReturn(organisationParents);
+
+        when(courseService.getMandatoryCourses(organisationParents, status, PageRequest.of(0, 20)))
+                .thenReturn(expected);
 
-        when(courseRepository.findMandatory(eq(department), eq(status), any(Pageable.class)))
-                .thenReturn(new ArrayList<>(Collections.singletonList(course)));
-        when(courseService.getOrganisationParents(eq(department))).thenReturn(new ArrayList<>(Collections.singletonList(department)));
         mockMvc.perform(
                 get("/courses/")
                         .param("department", "department1")
                         .param("mandatory", "true")
                         .with(csrf()))
                 .andExpect(status().isOk())
-                .andExpect(jsonPath("$.results[0].id", equalTo(course.getId())));
+                .andExpect(jsonPath("$.results[0].audiences[0].departments[0]", equalTo(course1.getAudiences().iterator().next().getDepartments().iterator().next())))
```

Comment:
This line is confusing.  Can you just check that the audiences contains the expected departments of dh and hmrc.
If the line length is still too long, put on 2 lines.
For example a hamcrest matcher or assertJ contains.
See 
https://www.baeldung.com/hamcrest-collections-arrays